Add all items counter on the Homepage

Wahaj-Ali opened the pull request:
in this commit
- created a faction that counts all shows on homepage and show it on the homepage.

Wahaj-Ali committed:
Add all items counter on the Homepage

in this commit
- created a faction that counts all shows on homepage and show it on the homepage.

Lucas-Erkana reviewed (changes requested):
❌ ESlint error
![image](https://user-images.githubusercontent.com/41428579/210708484-fbd8a132-48ab-4cf2-bacd-bbeac24a3804.png)
Kindly change 
![image](https://user-images.githubusercontent.com/41428579/210710436-692954c9-8099-42dd-a2bb-c5097a49c4d6.png)
to 
![image](https://user-images.githubusercontent.com/41428579/210710556-fab85585-fc73-4479-a9b8-ed1821af11bb.png)


❌ Each counter should be implemented as a separate module.
❌ A counter function should look for specific DOM elements
Kindly have a look at [this](https://www.w3schools.com/jsref/prop_element_childelementcount.asp) to count DOM elements
✅ counting based on what is actually displayed on the page and display on the screen

Wahaj-Ali committed:
used separate module for itemscounter.

in this commit
- created a module with a function to count all the items using specific DOM element.

Wahaj-Ali committed:
Update itemscounter.js

removed linter errors

Lucas-Erkana reviewed (approved):
✅ no linter errors
✅ Each counter should be implemented as a separate module.
✅ A counter function should look for specific DOM elements
✅ counting based on what is actually displayed on the page and display on the screen
